service/controller/key/key.go

```diff
@@ -38,12 +38,12 @@ const (
 	AnnotationEtcdDomain        = "giantswarm.io/etcd-domain"
 	AnnotationPrometheusCluster = "giantswarm.io/prometheus-cluster"
 
-	LabelApp           = "app"
-	LabelCluster       = "giantswarm.io/cluster"
-	LabelCustomer      = "customer"
-	LabelManagedBy     = "giantswarm.io/managed-by"
-	LabelOrganization  = "giantswarm.io/organization"
-	LabelVersionBundle = "giantswarm.io/version-bundle"
+	LabelApp             = "app"
+	LabelCluster         = "giantswarm.io/cluster"
```

Comment:
I'm leaving the comment in the file so discussion is easier because of the github ui.

Why do you have the `flatten` branch? What we were doing so far was creating `legacy` branch out of `master` and switching draughstman deployment from there. Do you have different plan?

Reply:
I was following the same approach I saw [people used for kvm-operator](https://github.com/giantswarm/kvm-operator/pull/817). The final step will be a PR merging the `flatten` branch into `master`. Is there a problem with that approach?

Reply:
I didn't notice how it was done in kvm-operator. If this approach is more convenient that's good.